### PR TITLE
Add validity icons in the TreeEditor

### PR DESCRIPTION
**Pull request by martinRenou**
Adds valid/invalid icons to show error statuses of the various components of the workflow.
At the moment, it just provides the basic infrastructure for adding the icons, and the icons themselves. 
No actual validation takes place.

### CODE REVIEW

File: force_wfmanager/left_side_pane/workflow_settings.py [hunk 1]
```diff
@@ -90,77 +90,95 @@ def delete_entity_handler(self, editor, object):
     action='handler.delete_entity_handler(editor, object)'
 )
 
+
+class WorkflowElementNode(TreeNode):
+    """ Custom TreeNode class for worklow elements """
+    def get_icon(self, object, is_expanded):
+        if object.valid:
```
Comment (stefanoborini) [inline force_wfmanager/left_side_pane/workflow_settings.py]: you can write `return 'icons/valid.png' if object.valid else 'icons/invalid.png'`

File: force_wfmanager/left_side_pane/icons/license.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+The icons are mostly derived work from other icons. As such they are
```
Comment (stefanoborini) [inline force_wfmanager/left_side_pane/icons/license.txt]: Probably better to have it as a section of the LICENSE.txt file.

File: force_wfmanager/left_side_pane/icons/license.txt [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+----------------------------------------------------------------------------
+Enthought               BSD 3-Clause              LICENSE.txt
+
+Unless stated in this file, icons are the work of Enthought, and are
+released under a 3 clause BSD license.
+
+Files and orginal authors:
+----------------------------------------------------------------------------
+force-wfmanager/force_wfmanager/left_side_pane/icons:
+                            all | Enthought
```
Comment (stefanoborini) [inline force_wfmanager/left_side_pane/icons/license.txt]: I am unsure if the size of the invalid icon is too big compared to the available space.

File: force_wfmanager/left_side_pane/workflow_settings.py [hunk 4]
```diff
@@ -90,77 +90,95 @@ def delete_entity_handler(self, editor, object):
...
+            return 'icons/valid.png'
+        return 'icons/invalid.png'
+
+
 tree_editor = TreeEditor(
     nodes=[
         # Root node "Workflow"
-        TreeNode(node_for=[WorkflowModelView],
```
Comment (stefanoborini) [inline force_wfmanager/left_side_pane/workflow_settings.py]: this is just reformatting it seems, right?

Comment (martinRenou) [inline force_wfmanager/left_side_pane/workflow_settings.py]: Yes, just reformatting for the `TreeNode` instances

File: force_wfmanager/left_side_pane/icons/license.txt (hunk 3 above)
Comment (martinRenou) [inline force_wfmanager/left_side_pane/icons/license.txt]: The `TreeEditor` seems to render the them with the same size automatically

**Pull request merged**